cypress/integration/plugins/security-dashboards-plugin/auth_test.cy.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Log in via OIDC', () => {
+  const login = 'admin';
```

Comment:
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress.json#L16

I think you can pull from the cypress env variable for these as well unless you want to have a separate cypress variable defined as like `OIDC_USERNAME`

Reply:
@jakubp-eliatra can You take care of frontend part?